tests/jballerina-unit-test/src/test/resources/testng-new-parser.xml

```diff
@@ -130,7 +130,8 @@
             <package name="org.ballerinalang.test.taintchecking.*"/>
             <package name="org.ballerinalang.test.functions.*"/>
             <package name="org.ballerinalang.test.javainterop.*"/>
-            <package name="org.ballerinalang.test.execution.*" />
+            <package name="org.ballerinalang.test.execution.LargeInitBuildTest" />
+<!--            <package name="org.ballerinalang.test.execution.ModuleExecutionFlowTests" />-->
```

Comment:
This is still failing for the old parser test suite. Can you please update `testng-jballerina.xml` also?

Reply:
Disabled in `testng-jballerina.xml`